actors/market/src/deal.rs

```diff
@@ -66,6 +66,15 @@ impl<'de> Deserialize<'de> for Label {
     }
 }
 
+impl Label {
+    pub(crate) fn len(&self) -> usize {
```

Comment:
Eh, just make it public. And add an `is_empty` method because clippy is going to complain.